test/main.spec.js

```diff
@@ -0,0 +1,14 @@
+import test from 'tape'
+import addTwoNumbers from './../src/main'
```

Comment:
I vote for adding a new setting to the local eslint config (not the commercetools-eslint-config), because this feels very application specific.